index.rst

```diff
@@ -10,20 +10,849 @@
+The aspect pertaining specifically to the FAFF charge is what the observer is required to look at during the process, including both images and/or alarms.
+The details of how Daytime Calibration is performed is being documented in `DMTN-222 <https://DMTN-222.lsst.io>`_ and will not be repeated as a new FAFF use-case.
+
+In short, a SAL script is launched by the observer to acquire a daytime set of calibrations.
+This SAL script launches an OCPS-based processing of the images, but the ScriptQueue does not block on the processing awaiting the final analysis.
+Currently, if the process fails then no alert is generated automatically.
+However, as will be discussed in the following sections, a Watcher [#]_ alarm will be setup to listen and alert users (via LOVE) in the event of a catastrophic failure in the analysis which the observer could do something about (e.g. the shutter did not open and the flats have no signal).
+How the observer responds to the alert is currently being discussed.
+Presumably, this will use a parameterized notebook that will allow an observer to better understand the issue.
+Any viewing of the raw frames themselves will utilize the Camera Visualization Tool.
+
+In the case where a more complex issue arises (e.g., a 2% increase in bad pixels is observed), this is addressed by the calibration team offsite and is not immediately reported to the summit team.
+When the calibrations used on the summit need to be updated, this is the role of the calibration scientist and is not the responsibility of the observer.
+Furthermore, this cadence is expected to be slow (months) and is therefore outside the scope of this charge.
+
+.. [#] The `Watcher CSC <https://ts-watcher.lsst.io/>`_ is provided a list of "rules" that it ensures the system is always obeying. If a rule is violated, such as a temperature going out of specification, the an alert or alarm is issued to the observer via the LOVE interface. The alarm stays in place until the rule is no longer violated and the original alert has been acknowledged. The Watcher is not able to perform analyses and only evaluates simple conditions.
+
+.. _Deliverable 2:
+
+Deliverable 2: Rapid Analysis Calculated Metrics
+------------------------------------------------
+
+.. note::
+
+   The deliverable description from the `charge`_ has been directly copied here to ease readability.
+
+
+   2. (`SITCOM-180`_, `SITCOM-173`_) Define which metrics, analyses and artifacts must be calculated and on what timescale they must be evaluated and reported to support commissioning/operations.
+
+      This is to evaluate if a "rapid processing" of data is required, what specific calculations are required.
+      This list should include the relevant camera specific calculations (which are currently performed by the EO testing data reduction).
+      This is expected to inform the answer to the next charge task.
+
+
+It is important to note that the charge question above refers to "rapid processing."
```

Comment:
"rapid processing." -> "rapid processing".

Reply:
Periods and commas go inside of quotes.

Reply:
oh! That is news to me.. cool..